falkordb/asyncio/falkordb.py

```
@@ -82,6 +90,18 @@ def __init__(
                            retry=retry, redis_connect_func=connect_func,
                            credential_provider=credential_provider,
                            protocol=protocol)
+        
+        if Is_Cluster(conn):
```

Ruff (0.8.2)

`Is_Cluster` may be undefined, or defined from star imports

(F405)